src/main/java/uk/gov/ons/ctp/response/sample/endpoint/SampleEndpoint.java

```diff
@@ -106,6 +107,32 @@ public SampleEndpoint(SampleService sampleService, MapperFacade mapperFacade) {
     return ResponseEntity.created(URI.create(newResourceUrl)).body(sampleUnitsRequest);
   }
 
+  @RequestMapping(
+      value = "/sampleunitsize",
+      method = RequestMethod.POST,
```

Comment:
Should this not be a get? No state is being altered here

Reply:
it's true it's more like a GET, but GET means a resource request (i.e. specified by URL). We are supplying a body... in fact we are POSTing a body - it's a bi-directional operation.

I thought long and hard about making it a GET. In fact I even tried it and found that a GET with a body is rejected by Jetty.

Reply:
Completely missed that. Fair play